agenta-backend/agenta_backend/services/app_manager.py

```diff
@@ -183,6 +184,64 @@ async def update_variant_image(
     await start_variant(app_variant_db)
 
 
+async def update_last_modified_by(
```

Comment:
**_For now_**
It's ok that this function works the way it does, because there is no real boundary between domains and because domain-relationships will evolve in the short-to-mid term.

**_Eventually_**
This has to happen from within the Applications domain and other domains shouldn't even know that this is happening.

Reply:
@aybruhm @mmabrouk -- just a note for future us.

Reply:
Yes, I agree with you.